checksetup.pl

```diff
@@ -53,7 +53,8 @@ BEGIN
                      'cpanm:s', 'check-modules',
                      'make-admin=s', 'reset-password=s', 'version|V',
                      'no-assets',
-                     'no-permissions|p');
+                     'default-localconfig',
+                     'no-database', 'no-permissions|p');
```

Comment:
If 'no-database' was worth combining with 'no-permissions', should 'no-assets' be rolled into this single line as well?